src/datasources/config-api/config-api.service.ts

```diff
@@ -38,10 +39,29 @@ export class ConfigApi implements IConfigApi {
 
   async getChain(chainId: string): Promise<Chain> {
     try {
-      const key = `${chainId}_chain`;
+      const cacheKey = `${chainId}_chain`;
       const field = '';
       const url = `${this.baseUri}/api/v1/chains/${chainId}`;
-      return await this.dataSource.get(key, field, url);
+      return await this.dataSource.get(cacheKey, field, url);
+    } catch (error) {
+      throw this.httpErrorFactory.from(error);
+    }
+  }
+
+  async getSafeApps(
+    chainId: string,
```

Comment:
This should not be required – it should be possible to retrieve the collection of Safe Apps for every chain.

Even though it is required at the route level I think we should match the datasource with what is possible to get from it. Wdyt?

Reply:
Yes, I think it's better to match the datasource at the datasource level, and let the param as required at the route level. I'll change it, thanks 👍🏻

Reply:
Changed in [32a5f1eb7e](https://github.com/5afe/safe-client-gateway-nest/commit/32a5f1eb7eb7e1e2b02d5e785aba5335a72d3cac)